vagrant/provisioning/roles/httpd/templates/ldap-auth-restricted-solr

```diff
@@ -5,10 +5,12 @@ RewriteEngine On
 #LDAPReferrals Off
 # Perform LDAP lookup for user to check group membership
 AuthLDAPUrl "{{ ldap_url }}/{{ ldap_base }}?{{ ldap_remote_user_attribute | default('sAMAccountName') }}?sub?(objectClass=*)"
-AuthLDAPBindDN "{{ ldap_base }}"
+AuthLDAPBindDN "{{ ldap_base | regex_search('(?<=\CN=)(.*?)(?=\,)') }}@{{ ldap_user_domain }}"
```

Comment:
Is AuthLDAPBindDN supposed to be a distinguished name, or a user id?  ldap_base obviously is not a user id at all ("OU=EEOC,OU=Clients,DC=appdev,DC=armedia,DC=com").  Should you be using ldap_bind_user here instead?

Reply:
AuthLDAPBindDN shoud be in the following form for ex. "svc.ark.core.ldap@appdev.armedia.com"...
The first string before @ is the value of CN, and instead of using new fact for CN, i'm using regex for extracting only CN from ldap_bind_user. 
You are right, instead of ldap_base, it should be ldap_bind_user